### PR TITLE
Lint Test Fixes

### PR DESCRIPTION
**Pull request by codingmusician**
### 🗒️ Description
My Innovation Day Work: Fixing the failing lint tests for TEC (and other plugins)

### CODE REVIEW

File: src/resources/js/event-editor.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ var tribe_events_event_editor = tribe_events_event_editor || {};
 	 *
 	 * @since 6.0.1
 	 */
-	obj.init = () => {
+	obj.init = function(){
```
Comment (bordoni) [inline src/resources/js/event-editor.js:47]: Lets keep the other one.

**Review by bordoni — CHANGES_REQUESTED** [review]

**Comment by bordoni** [comment]
Do you mind linking all of these PRs?

File: src/resources/js/event-editor.js (hunk 1 above)
Comment (codingmusician) [reply src/resources/js/event-editor.js:47]: @bordoni I disagree. All other functions in this file are not arrow functions, plus we would have to explicitly ignore the `Unexpected arrow-function expression` error that `eslint` raises.

Comment (codingmusician) [reply src/resources/js/event-editor.js]: @bordoni Do you mind taking a look at this again? I'd like to get it merged into the next MR if possible and make this test required.

**Review by bordoni — APPROVED** [review]

**Review by Camwyn — APPROVED** [review]

**Pull request merged**